aspnetcore/blazor/hosting-models.md

```diff
@@ -23,7 +25,7 @@ In a traditional server-rendered app, opening the same app in multiple browser s
 
 On the client, the Blazor script (`blazor.server.js`) establishes the SignalR connection with the server. The script is served to the client-side app from an embedded resource in the ASP.NET Core shared framework. The client-side app is responsible for persisting and restoring app state as required. 
 
-The Blazor Server hosting model offers several benefits:
+You should use the Blazor Server hosting model for the following benefits:
```

Comment:
The updated language feels a bit clunky to me. I prefer the original in spite of SEO considerations.